app/code/Magento/Catalog/Model/View/Asset/Placeholder.php

```diff
@@ -152,10 +161,13 @@ public function getFilePath()
         if ($this->filePath !== null) {
             return $this->filePath;
         }
+
+        $storeId = $this->request->getParam('filters')['store_id'];
```

Comment:
I guess we need to make sure the value of `filters` parameter is an array and `store_id` key exists in order to avoid potential exceptions.

Reply:
I attempted to do this, let me know what you think.
Thanks.

Reply:
Sure, but I would simplify the condition:
```$storedId = isset($filters['store_id']) ? $filters['store_id'] : null;```